spec/definitions/boxes/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+group :kvm do
+  gem "ruby-libvirt"
+  gem "veewee", "~> 0.4.5"
+  gem "fog", "~> 1.27.0"
```

Comment:
I know it's not the case for you but I keep getting this issue with fog 1.27:

    warning: duplicated key at line 196 ignored: :openstack_region (...). (Pennyworth::ExecutionFailed)

Just by upgrading `fog` and `fog-core` to `~> 1.28.0` I get rid of the warning, it's documented here: `https://github.com/fog/fog/issues/3391`